chat-client/src/client/mynahUi.ts

```diff
@@ -223,6 +224,12 @@ export const createMynahUi = (
                 ],
                 ...(disclaimerCardActive ? { promptInputStickyCard: disclaimerCard } : {}),
             }
+
+            const tabStore = mynahUi.getTabData(tabId).getStore()
+
+            if (!tabStore?.tabMetadata || !tabStore.tabMetadata.openTabKey) {
```

Comment:
What are we doing here? Why? Can you please add description in comment in the code, because this code intent is not super clear.